packages/tokens/build.js

```diff
@@ -134,20 +142,26 @@ StyleDictionary.registerFormat({
   name: 'typescript/es6-declarations/spacing',
   formatter: function (dictionary) {
     let declaration = 'export declare const spacing: {\n'
-    dictionary.allProperties.forEach((token) => {
+    let allValuesComment = '/**\n * '
+    dictionary.allProperties.forEach((token, index) => {
+      declaration += `  /** Value: ${token.value} */\n`
       declaration += `  ${token.name}: number;\n`
+      allValuesComment += `${index !== 0 ? ', ' : ''}${token.name.replace('vadsSpace', '')}: ${token.value}`
```

Comment:
Might be somewhat more legible in a single line with something like:
```suggestion
      allValuesComment += `${index !== 0 ? ' | ' : ''}${token.name.replace('vadsSpace', '')}: ${token.value}`
```
to get
`None: 0 | 2xs: 4 | Xs: 8 | Sm: 12,`
or something to more clearly delineate than comma separation alongside the values themselves using colon separation.